CHANGELOG.md

```diff
@@ -1,3 +1,10 @@
+## 6.0.0
```

Comment:
Is the change indicating a 'breaking change' due to no longer setting of a default tag?

Reply:
Yes that's the breaking change.

Now that you're asking this, I wonder. Should we mark what is the breaking change in our changelogs? And change the guidelines accordingly?

Reply:
I mean with a standard way. Perhaps prepend the line(s) this with `[Breaking]` or something

Reply:
Yeah, I think we should have a standard way of specifying breaking changes. [Historically](https://github.com/search?l=Markdown&q=org%3Alogstash-plugins+breaking&type=Code), it looks like the most common way of indicating this has been to add `Breaking: ` before each breaking change in the changelog. I don't mind doing it either this way, or using your suggestion, but we should choose one, stick to it, and update the contributing guidelines to reflect our decision.

WDYT?

cc @jsvd

Reply:
+1 on using either [Breaking] or Breaking (maybe `[BREAKING]`?), just create a PR to add that extra guideline on the CONTRIBUTORS.md in elastic/logstash